javascript/stacksAndQueues/fifoAnimalShelter/README.md

```diff
@@ -0,0 +1,14 @@
+# Challenge Summary
+First-in, First out Animal Shelter.
+
+## Challenge Description
+Create a class called AnimalShelter which holds only dogs and cats. The shelter operates using a first-in, first-out approach.
+- Implement the following methods:
+  - enqueue(animal): adds animal to the shelter. animal can be either a dog or a cat object.
+  - dequeue(pref): returns either a dog or a cat. If pref is not "dog" or "cat" then return null.
+
+## Approach & Efficiency
+My team and I worked on the whiteboard for an hour. Big O for enqueue is O(1) and 0(n) for dequeue.
+
+## Solution
+![FiFo Animal Shelter](fifoAnimalShelter.png)
```

Comment:
Off to a great start! Please take your time with this! I will come back and regrade this once you have worked out code and tests!